playwright/tests/script-runner/file-menu.spec.ts

```diff
@@ -40,37 +40,37 @@ test('open a file', async ({ page, utils }) => {
   await page.locator('text=Open File').click()
   await expect(page.getByText('INST2', { exact: true })).toBeVisible()
   await utils.sleep(100)
-  await page.locator('[data-test=file-open-save-search]').type('dis')
+  await page.locator('[data-test=file-open-save-search] input').fill('dis')
   await utils.sleep(100)
-  await page.locator('[data-test=file-open-save-search]').type('con')
+  await page.locator('[data-test=file-open-save-search] input').fill('con')
   await utils.sleep(100)
-  await page.locator('[data-test=file-open-save-search]').type('nect')
+  await page.locator('[data-test=file-open-save-search] input').fill('nect')
   await utils.sleep(100)
```

Comment:
The idea here was to type it like the user would which is what the incremental type with delay is. But I see in the script-runner/api.spec that you just removed it all and did a simple fill. If that works here great.